includes/html/graphs/device/bigip_ltm_allpm_currconns.inc.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/*
+ * LibreNMS module to display F5 LTM Virtual Server Details
+ *
+ * Copyright (c) 2016 Aaron Daniels <aaron@daniels.id.au>
```

Comment:
Hi
Could you please correct the copyright on files ?
Thanx

Reply:
Thanks! Forgot about that part